statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1206,12 +1205,6 @@ def test_conf_int(self):
     def test_jac(self):
```

Comment:
I'm getting close to a "LGTM".  I'm hoping we can take care of these `test_jac`s and the commented-out `test_cov_params` while this module is fresh in your head.  Any thoughts?